app/controllers/admin/analytics_controller.rb

```diff
@@ -8,9 +8,9 @@ class AnalyticsController < ApplicationController
     layout 'admin'
 
     def index
-      @ui_analytics = get_ui_analytics
-      @total_homepage_views = total_homepage_views(@ui_analytics)
-      @new_homepage_views = new_homepage_views(@ui_analytics, @start_date, @end_date)
+      @total_homepage_views = get_ui_analytics['total_homepage_views']
+      @new_homepage_views = get_ui_analytics['new_homepage_views']
```

Comment:
Even though this is now a faster call, we still shouldn't fetch the json twice, grab it once, store it locally to this method, and then hash access it twice.

Reply:
done